studio/styles/grid.scss

```diff
@@ -527,6 +527,22 @@
   @apply border-scale-300;
 }
 
+/*
+  Cell copy action
+*/
+
+.sb-grid-select-cell__copy-action {
+  @apply hidden mr-1;
+}
+
+.sb-grid-select-cell__formatter:hover .sb-grid-select-cell__copy-action {
+  @apply block opacity-50;
+}
+
+.sb-grid-select-cell__copy-action:hover {
+  @apply opacity-100;
+}
+
```

Comment:
Why create its own css class, when you can just put this into the className for that div.

Reply:
Thanks @Isaiah-Hamilton we updated PR to use pure tailwindcss classes instead of creating new global css class.